### DISCUSSION TITLE
Is Sygic app for Android still supports BLE-HUD?

### DISCUSSION THREAD

**Discussion by Riddickos** [alexanderlavrushko/BLE-HUD-navigation-ESP32#21] [General]
I'm trying to enable BLE HUD in Sygic for Android but looks like in latest verison it is not supported anymore.

**Comment by alexanderlavrushko** [comment]
Sygic app for Android never supported BLE HUD, and now it still doesn't. Maybe in future it will, not sure about that so far.
